packages/server/lib/controllers/files.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   handleFiles (req, res, config) {
     debug('handle files')
 
-    return specsUtil.find(config)
+    return specsUtil.default.findSpecs(config)
```

Comment:
Not really ideal to need to call `default` but not having the `default` export means some tests need reworking slightly, and I want to keep the changes to a minimum, so I just did it like this.